src/components/DropDownAddressSelector/DropDownAddressSelector.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="drop-down-address-selector">
+    <div
+      :class="dropdownOpen ? 'dropdown-open' : ''"
+      class="dropdown-input-box">
+      <input
+        v-model="selectedAddress"
+        type="text"
+        placeholder="Please enter the address"
+        @focus="dropdownOpen = false">
+      <div
+        v-if="!validAddress"
+        class="blockie-place-holder-image"/>
+      <div
+        v-if="validAddress"
+        class="selected-address-blockie">
+        <blockie
+          :address="selectedAddress"
+          width="30px"
+          height="30px"/>
+      </div>
+      <div
+        class="dropdown-open-button"
+        @click="dropdownOpen = !dropdownOpen">
+        <i
+          v-if="!dropdownOpen"
+          class="fa fa-chevron-down"
+          aria-hidden="true"/>
+        <i
+          v-if="dropdownOpen"
+          class="fa fa-chevron-up"
+          aria-hidden="true"/>
+      </div>
+    </div>
+    <div
+      v-if="dropdownOpen"
+      class="dropdown-list-box">
+      <ul>
+        <li
+          v-for="addr in addresses"
+          :key="addr.key"
+          @click="listedAddressClick(addr)">
+          <div class="list-blockie">
+            <blockie
+              :address="addr"
+              width="30px"
+              height="30px"/>
+          </div>
+          <p class="listed-address">{{ addr }}</p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+import Blockie from '@/components/Blockie';
+import web3 from 'web3';
+
+export default {
+  components: {
+    blockie: Blockie
+  },
+  data() {
+    return {
+      selectedAddress: '',
+      validAddress: false,
+      dropdownOpen: false,
+      addresses: [
```

Comment:
Probably better to move this to props since it looks like this component will be set to receive the addresses.